test/dashboard-e2e/api/api-helpers.js

```diff
@@ -1,17 +1,26 @@
 import superagent from 'superagent';
 
 export class ApiHelpers {
-    constructor(apiUrl) {
+    constructor(apiUrl, cloudContext='', bearerToken='') {
         this.API_URL = apiUrl;
+        this.CLOUD_CONTEXT = cloudContext;
+        this.BEARER_TOKEN = bearerToken;
```

Comment:
Properties should be camel case:
```suggestion
        this.apiUrl = apiUrl;
        this.cloudContext = cloudContext;
        this.bearerToken = bearerToken;
```